### PR TITLE
Emit "newAlarm" when alarm is undefined

### PR DESCRIPTION
**Pull request by maneetgoyal**
- Fixes https://github.com/node-opcua/node-opcua/issues/793

### CODE REVIEW

**Comment by maneetgoyal** [comment]
@erossignon Can you please suggest how to fix the CI failure?

**Comment by erossignon** [comment]
The CI failure : in fact the test passes .. only publishing coverage report to coveralls website fails...

File: packages/node-opcua-client/source/alarms_and_conditions/client_alarm_list.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export class ClientAlarmList extends EventEmitter implements Iterable<ClientAlar
             const key = this.makeKey(conditionId.value, eventType.value);
             const newAlarm = new ClientAlarm(eventField);
             this._map[key] = newAlarm;
+            this.emit("newAlarm", newAlarm);
```
Comment (erossignon) [inline packages/node-opcua-client/source/alarms_and_conditions/client_alarm_list.ts:59]: A unit test is requested

**Comment by maneetgoyal** [comment]
> The CI failure : in fact the test passes .. only publishing coverage report to coveralls website fails...

It might be this issue: https://github.com/lemurheavy/coveralls-public/issues/632. Will give it another go.

**Review by erossignon — CHANGES_REQUESTED** [review]
Please add a unit test

File: packages/node-opcua-client/source/alarms_and_conditions/client_alarm_list.ts (hunk 1 above)
Comment (maneetgoyal) [reply packages/node-opcua-client/source/alarms_and_conditions/client_alarm_list.ts:59]: On it 👍

Comment (maneetgoyal) [reply packages/node-opcua-client/source/alarms_and_conditions/client_alarm_list.ts:59]: Done, please check.

**Pull request closed without merging**